src/main/java/org/lantern/AppIndicatorTray.java

```diff
@@ -258,10 +257,9 @@ public boolean isActive() {
     }
 
     @Subscribe
-    public void onConnectivityStateChanged(final ProxyConnectionEvent csce) {
-        final ConnectivityStatus cs = csce.getConnectivityStatus();
-        LOG.debug("Got connectivity state changed {}", cs);
-        onConnectivityStatus(cs);
+    public void onConnectivityStateChanged(final ProxyConnectedEvent csce) {
+        LOG.debug("Got ProxyConnectedEvent");
+        onConnectivityStatus(ConnectivityStatus.CONNECTED);
```

Comment:
We no longer have different proxy connectivity statuses.  Once the proxy is connected, it is connected.